systemd/systemd.go

```diff
@@ -301,7 +301,20 @@ func MockJournalctl(f func(svcs []string, n int, follow, namespaces bool) (io.Re
 	}
 }
 
+// MountUnitType is an enum for the supported mount unit types.
+type MountUnitType int
+
+const (
+	// RegularMountUnit is a mount with the usual dependencies
+	RegularMountUnit MountUnitType = iota
+	// BeforeModulesLoadMountUnit mounts things before kernel modules are
+	// loaded either by udevd or by systemd-modules-load.service.
+	BeforeModulesLoadMountUnit
```

Comment:
udev could also do things with firmware which will also ship in kernel-modules component, so maybe BeforeDriversLoad... ?